src/components/BirthdayPicker.tsx

```diff
@@ -2,30 +2,51 @@ import DatePicker from "react-datepicker";
 import "react-datepicker/dist/react-datepicker.css";
 import {useState} from "react";
 import "./birthday.scss";
+import {registerLocale} from "react-datepicker";
+import {enGB} from "date-fns/locale";
+import {FieldInputProps, FormikProps} from "formik";
 
-const BirthdayPicker = () => {
-  const [value, setValue] = useState<Date | null>(null);
-  const fieldHandler = (date: Date) => {
-    setValue(date);
+registerLocale('enGB', enGB)
```

Comment:
move this line

registerLocale('enGB', enGB)

 into BirthdayPicker component and test that it works
 
 It is logically better to have this logic inside component